docs/extensibility/resources.md

```diff
@@ -26,7 +26,7 @@ The `general` section is required and contains basic information about the resou
 - **scope** - either `namespace` or `cluster`. Defaults to `cluster`.
 - **urlPath** - path fragment for this resource used in the URL. Defaults to pluralized lowercase **kind**. Used to provide an alternative URL to avoid conflicts with other resources.
 - **defaultPlaceholder** - to be shown in place of empty resource leaves. Overridden by the widget-level **placeholder**. Defaults to `-`.
-- **description** - displays a custom description on the resource list page. It can contain links. If the translation section has a translation entry with the ID that is the same as the **description** string, the translation is used.
+- **description** - displays a custom description on the resource list page. It can contain links. If the `translations` section has a translation entry with the ID that is the same as the **description** string, the translation is used.
```

Comment:
This looks to me like it should be in bold, not in code font. 
If it's a UI element or a parameter/key (and not a value for the given key), it should be in bold. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-bold-font-for-these-items) for reference.
```suggestion
- **description** - displays a custom description on the resource list page. It can contain links. If the **translations** section has a translation entry with the ID that is the same as the **description** string, the translation is used.
```